checkout-service/client/src/components/AddToListDropDown.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import faker from 'faker';
+
+const AddToListDropDownWrapper = styled.div`
+  background-color: #E7E9EC;
+  border: 1px solid;
+  border-color: #adb1b8 !important;
+  font-family: "Amazon Ember",Arial,sans-serif;
+  font-size: 13px;
+  line-height: 29px;
+  border: 1px solid;
+  margin-top: 13px;
+  height: 29px;
+  border-radius: 3px;
+  cursor: pointer;
+`;
+
+const AddToListSection = styled.span`
+  width: 83%;
+  display: inline-block;
+  border-right: 1px solid;
+  border-color: #adb1b8;
+  height: 29px;
+  padding-left: 5%;
+  &:hover {
+    background-color: #e2e5ea;
+  }
+`;
+
+const DropDownArrowSection = styled.span`
+  position: absolute;
+  width: 26px;
+  height: 29px;
+  &:hover {
+    background-color: #e2e5ea;
+  };
+  &:focus {
+    box-shadow: 0 0 3px 2px rgb(228 121 17 / 50%);
+    outline: 0;
+  };
+`;
+
+const DropDownArrowIcon = styled.i`
+  position: absolute;
+  margin: 10px 0 0 8px;
+  height: 8px;
+  width: 10px;
+  background-image: url("https://m.media-amazon.com/images/S/sash/McBZv0ZvnbehkIx.png");
+  background-position: -7px -102px;
+  border: none;
+`;
+
+const DropDownListContainer = styled("div")``;
+
+const DropDownList = styled("ul")`
+  padding: 0;
+  margin: 0;
+  background: #ffffff;
+  border: 1px solid #e5e5e5;
+  box-sizing: border-box;
+  font-size: 1.3rem;
+  font-weight: 500;
+  &:first-child {
+    margin-top: 0.1em;
+  }
+`;
+
+const ListItem = styled("li")`
+  list-style: none;
+  height: 29px;
+  margin: 0.2em 0 0.2em 0;
+  &:hover{
+    background-color: #e2e5ea;
+  }
+`;
+
+const ListItemImgWrapper = styled.span`
+  height: 29px;
+  margin: 0 8px 0 8px;
+`;
+
+const ListItemImg = styled.img`
+  height: 22px;
+  width: 30px;
+
+`;
+
+const ListItemTextWrapper = styled.div`
+  height: 29px;
+  display: inline-block;
+`;
+
+const ListItemLabel = styled.span`
+  font-size: 12px;
+  height: 14px;
+  line-height: 14px;
+  display: block;
+`;
+
+const ListItemStatus = styled.span`
+  font-size: 12px;
+  height: 14px;
+  line-height: 14px;
+  display: block;
+  color: #565959;
+`;
+
+const generateListItems = () => {
+  let randomNumOfLists = Math.floor(Math.random() * (5 - 1) + 1);
+  let listStatus = ['public', 'private'];
+  let listItems = [];
+  for (let i = 0; i < randomNumOfLists; i++) {
+    let randomPrivateOrPublicIndex = Math.floor(Math.random() * Math.floor(2));
+    listItems.push(<ListItem><ListItemImgWrapper><ListItemImg src={faker.image.imageUrl()}></ListItemImg></ListItemImgWrapper><ListItemTextWrapper><ListItemLabel>{faker.random.word()}</ListItemLabel><ListItemStatus>{listStatus[randomPrivateOrPublicIndex]}</ListItemStatus></ListItemTextWrapper></ListItem>);
```

Comment:
That's right! The name is in here `<ListItemLabel>{faker.random.word()}</ListItemLabel>` since I'm not persisting any user data, I'm just using a random faker word for the name.

Reply:
Got it.